core/src/test/resources/HistoryServerExpectations/excludeOnFailure_for_stage_expectation.json

```diff
@@ -631,6 +642,7 @@
       "taskId" : 4,
       "index" : 2,
       "attempt" : 1,
+      "partitionId": -1,
       "launchTime" : "2018-01-09T10:21:18.943GMT",
       "duration" : 16,
```

Comment:
Since we expect this to default to `-1`, revert changes to this (and similar other) file ?
Similarly for Partition Id = -1 from json files as well.

Reply:
These json file is generated by rest api. Unless we do not want to show  partitionId with history data, this still needs to be modified.

Reply:
Sounds good to keeping it as-is.
We need to validate that missing value defaults to -1 (for both json and level db) - to test upgrade case (this is dependent on Dongjoon's suggestions above on how to mitigate the issue though).